### PR TITLE
Added tests for Aside component

### PR DESCRIPTION
**Pull request by krolwofficial**
Added tests for Aside component
Added file to gitignore list

### CODE REVIEW

File: src/alto-ui/Aside/__tests__/Aside.test.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+import Aside from '../Aside';
+import CloseButton from '../../CloseButton';
+
+jest.mock('../../CloseButton', () => {
+  return () => {
+    return <div className="CloseButton" />;
+  };
+});
+
+describe('<Aside />', () => {
```
Comment (djdeimon) [inline src/alto-ui/Aside/__tests__/Aside.test.js]:
```suggestion
describe('Aside', () => {
```

File: src/alto-ui/Aside/__tests__/Aside.test.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+  const getWrapper = props => {
+    console.log(props);
```
Comment (djdeimon) [inline src/alto-ui/Aside/__tests__/Aside.test.js]: remove `console.log`

File: src/alto-ui/Aside/__tests__/Aside.test.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    return mount(<Aside {...props} />);
+  };
+
+  it('should return null if show props is falsy', () => {
```
Comment (djdeimon) [inline src/alto-ui/Aside/__tests__/Aside.test.js]:
```suggestion
  it('should return null if show props are falsy', () => {
```

File: src/alto-ui/Aside/__tests__/Aside.test.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    const props = {
+      show: false,
+    };
+    const wrapper = getWrapper(props);
+
+    expect(wrapper.find('.Aside__header')).toHaveLength(0);
```
Comment (barteksiudak) [inline src/alto-ui/Aside/__tests__/Aside.test.js]: assertion should be only one and looks like:
```
expect(wrapper.find('aside')).toHaveLenght(0);
```

File: src/alto-ui/Aside/__tests__/Aside.test.js [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+    expect(wrapper.find('.Aside__content').exists()).toBe(true);
+  });
+
+  it('should support show and a11yCloseLabel prop', () => {
+    const props = {
+      a11yCloseLabel: 'test',
+    };
+    const wrapper = getWrapper(props);
+
+    expect(wrapper.find(Aside).prop('show')).toBe(true);
```
Comment (barteksiudak) [inline src/alto-ui/Aside/__tests__/Aside.test.js]: DRY
```
const { show, a11yCloseLabel } = wrapper.find(Aside).props();
expect(show).toBe(true);
expect(a11yCloseLabel).toBe('test');
```

Comment (barteksiudak) [inline src/alto-ui/Aside/__tests__/Aside.test.js]: This test tests nothing!
This test should be on `<CloseButton />` not for main component.
```
const { open, a11yCloseLabel } = wrapper.find(CloseButton).props();
expect(open).toBe(true);
expect(a11yCloseLabel).toBe('test');
```

**Comment by barteksiudak** [comment]
Check if children are passed on correct place.
```
const wrapper = shallow(<Aside><div id="test">abc</div></Aside>);
expect(wrapper.find('.Aside__content').find({ id: 'test' )).toHaveLength(1);
```

File: src/alto-ui/Aside/__tests__/Aside.test.js (hunk 1 above)
Comment (krolwofficial) [reply src/alto-ui/Aside/__tests__/Aside.test.js]: Refactored

File: src/alto-ui/Aside/__tests__/Aside.test.js (hunk 2 above)
Comment (krolwofficial) [reply src/alto-ui/Aside/__tests__/Aside.test.js]: Removed

File: src/alto-ui/Aside/__tests__/Aside.test.js (hunk 3 above)
Comment (krolwofficial) [reply src/alto-ui/Aside/__tests__/Aside.test.js]: Changed to it('should no render if show `prop is` falsy', () => {

File: src/alto-ui/Aside/__tests__/Aside.test.js [hunk 6]
```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { mount } from 'enzyme';
+
+import Aside from '../Aside';
+import CloseButton from '../../CloseButton';
+
+jest.mock('../../CloseButton', () => {
```
[the remark this answers is not in the document]
Comment (krolwofficial) [reply src/alto-ui/Aside/__tests__/Aside.test.js]: Refactored

File: src/alto-ui/Aside/__tests__/Aside.test.js (hunk 4 above)
Comment (krolwofficial) [reply src/alto-ui/Aside/__tests__/Aside.test.js]: Refactored

File: src/alto-ui/Aside/__tests__/Aside.test.js (hunk 5 above)
Comment (krolwofficial) [reply src/alto-ui/Aside/__tests__/Aside.test.js]: Refactored

Comment (krolwofficial) [reply src/alto-ui/Aside/__tests__/Aside.test.js]: Refactored this test to test `<CloseButton/>` props

**Pull request merged**